### ISSUE TITLE
New preview link generation from CI

### ISSUE THREAD

**Issue by ailisp** [near/neardevhub-widgets#498]
After the new root setup in #294 , it's quite easily to develop and testing locally, which is excellent. However the ability to deploy a copy of the app under developer's personal account is broken, which makes PR review become more challenging. The reviewer has to clone the PR branch locally then try out. They can no longer just rely on the information from the PR. 

Ideally, we can have a CI which deploy the whole app to a new testnet account, so for most of PRs developer doesn't need to provide a preview link but CI will provide one. If the change is related to the contract data on mainnet, we ask the developer to submit a preview link where the app is deployed to his mainnet account.

**Comment by ori-near** [comment]
@frol @petersalomonsen can we close this ticket?

**Comment by petersalomonsen** [comment]
yes. we have several ways today of deploying previews, and also we have the test framework where we can record videos of changes.
